### ISSUE TITLE
2024_10_roadmap: DOC ability to see CLDs & DDD stocks side-by-side in MTL Manual

### ISSUE THREAD

**Issue by ljmoody** [lzim/teampsd#3255]
## AC:
- [ ] Dev for card 3190 is complete
- [ ] Functionality is tested and working correctly
- [ ] _MTL_ Manual updated with clear instructions on how to view CLDs and DDD stocks side-by-side

**Comment by ljmoody** [comment]
#adjudicate memo

- This card is essentially a duplicate of #3256, which will document the behavior of Sim UI pop-up windows in the _MTL_ Manual, once development of the feature (#3190) is complete.
- Revised point value to 1, for effort given to make/track this card. 
- Updated Epic and Milestone.
- Closing this card.
